### PR TITLE
#157694754 Add failed jobs page

### PR DESCRIPTION
**Pull request by muhozi**
#### What does this PR do?
- Add a tab in the links for the failed jobs(Issues)
- Add a page for the failed jobs
- Add tests for the failed job feature

#### Description of Task to be completed?
A user can click an issue tab/link and view listing of all their failed jobs
#### How should this be manually tested?
- Run the application and click the issues link to view the failed jobs/checks
- Test the added feature by running `./manage.py test hc.front.tests.test_my_checks`
#### What are the relevant pivotal tracker stories?
[#157694754](https://www.pivotaltracker.com/story/show/157694754)
#### Screenshot
screen shot 2018-06-07 at 20 36 38

### CODE REVIEW

File: static/js/issues.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        e.trigger.textContent = "copied!";
+        e.clearSelection();
+    });
+
+    clipboard.on('error', function (e) {
+        var text = e.trigger.getAttribute("data-clipboard-text");
+        prompt("Press Ctrl+C to select:", text)
+    });
+
+
```
Comment (david-kironde-deel) [inline static/js/issues.js]: why skip two lines here ?

**Review by david-kironde-deel — CHANGES_REQUESTED** [review]
Kindly add some screen shots for this feature.

File: static/js/issues.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                result += count + " " + unit.name + "s ";
+            }
+        }
+
+        return result;
+    }
+
+
+
+
```
Comment (david-kironde-deel) [inline static/js/issues.js]: Looks like too much space in-between.. 🤔

**Review by david-kironde-deel — APPROVED** [review]

**Comment by kzyangiro** [comment]
Good job Muhozi.

**Pull request merged**